src/PowerShellEditorServices/Workspace/Workspace.cs

```diff
@@ -202,6 +213,35 @@ public string GetRelativePath(string filePath)
             return resolvedPath;
         }
 
+        internal static string MapRemotePathToLocal(string remotePath, string connectionString)
+        {
+            string sessionTempPath =
+                Path.Combine(
+                    Path.GetTempPath(),
+                    "PSES-" + Process.GetCurrentProcess().Id,
+                    "RemoteFiles");
+
+            // The path generated by this code will look something like
+            // %TEMP%\PSES-[PID]\RemoteFiles\1205823508\computer-name\MyFile.ps1
```

Comment:
Do you think we need the PID?  One advantage of using a hash in the name is that if the file doesn't change, debugging it a second time doesn't create a new copy of the same file on my file system.  Then again, PowerShell script files usually aren't very big and folks are probably used to cleaning their temp folder from time to time.